geniza/corpus/views.py

```diff
@@ -65,7 +65,7 @@ def get_form_kwargs(self):
         return kwargs
 
     def get_queryset(self):
-        documents = DocumentSolrQuerySet()
+        documents = DocumentSolrQuerySet().filter(status="Public")
```

Comment:
We could make a `public` method on the local queryset similar to the wagtail `live` filter that applies this filter for us, since we're using it in multiple places. But I'm not sure how much it gets us.

Reply:
It used twice. Maybe on the third time?